hazelcast/src/main/java/com/hazelcast/spi/impl/UnmodifiableLazyList.java

```diff
@@ -16,16 +16,18 @@
 
 package com.hazelcast.spi.impl;
 
-import com.hazelcast.nio.ObjectDataInput;
-import com.hazelcast.nio.ObjectDataOutput;
 import com.hazelcast.internal.serialization.Data;
-import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
 import com.hazelcast.internal.serialization.SerializationService;
 import com.hazelcast.internal.util.UnmodifiableListIterator;
+import com.hazelcast.internal.util.collection.ImmutableLazyEntry;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
 
 import java.io.IOException;
 import java.util.AbstractList;
 import java.util.AbstractMap;
+import java.util.AbstractMap.SimpleImmutableEntry;
```

Comment:
this one is not used in my IDE. Why does not checkstyle complain?

Reply:
It's used this way:

```
                    item = new AbstractMap.SimpleImmutableEntry<>(key, value);
```

After removing the import it still compiles, I suppose it's checkstyle bug.

fa98d745bd3